Model/Db.php

```diff
@@ -62,22 +62,23 @@ public function insert($table_name, $columns = [])
      * @return array table data
      * 
      */
-    public function select($table_name, $filters = [], $limit = '', $offset = '' )
+    public function select($table_name, $filters = [], $limit = '', $offset = '', $order = '')
     {
         // Check if there is more than one filter to build the query correctly
-        if (count(array_filter($filters, null)) > 1){
-            $filters = array_filter($filters, null);
-            $filters = implode(' AND ', $filters);
-        }
-        else {
-            $filters = implode('', $filters);
-        }
+        $filtered_array = array_filter($filters);
 
+        if (!empty($filtered_array)) {
+            if (count($filtered_array) > 1) {
+                $filters = implode(' AND ', $filtered_array);
+            } else {
+                $filters = reset($filtered_array);
+            }
+        }
```

Comment:
Mmm creo que este chequeo de `$filters` no hace ninguna diferencia. Sería lo mismo hacer un `$filters = implode(' AND ', $filters);`. Porque `implode` sólo agrega la cadena " AND " si es que hay más de un elemento 🤔 . Por otro lado, se que no se agregó en esta PR, pero no es buena práctica redefinir una variable cambiando el tipo de dato. Yo creo que incluso se podría evitar volver a definir la variable con lo que comenté arriba